_sass/pages/_home.scss

```diff
@@ -17,6 +17,17 @@
   --light-text: hsl(0, 0%, 80%);
   --lighter-text: var(--white);
 
+  .highlight {
+    --md-code-fg-color: var(--light-text);
```

Comment:
I was expecting a sort of switch/include (light/dark mode) and not a series of settings (also, why the commented lines?)

Reply:
It's the easiest path. Dark mode is currently only used on the homepage, so I've focused the changes there. I was thinking of incorporating a general dark mode feature, at which point this part would be refactored.

Comments are leftovers from refactoring. I'll remove them.